openssl-dynamic/src/main/c/error.c

```diff
@@ -49,6 +49,19 @@ void tcn_ThrowException(JNIEnv *env, const char *msg)
 
 }
 
+void tcn_ThrowIllegalArgumentException(JNIEnv *env, const char *msg)
+{
+    jclass javaExceptionClass;
+
+    javaExceptionClass = (*env)->FindClass(env, "java/lang/IllegalArgumentException");
```

Comment:
I would prefer to cache all "static" classes ... but I agree its urgent and we can defer this.